spec/features/course_page_spec.rb

```diff
@@ -224,6 +226,20 @@ def js_visit(path, count=3)
       end
       expect(Course.last.passcode).to eq(previous_passcode)
     end
+
+    it 'allows edit in home_wiki_id' do
+      admin = create(:admin, id: User.last.id + 1)
+      login_as(admin)
+      allow(Features).to receive(:wiki_ed?).and_return(false)
```

Comment:
This might work. The typical way to do it would be to put this example inside a `context`. See feature_toggles_spec for a simple example of this.